### ISSUE TITLE
Reg\Register an Operation - New Entrant Operation has correct application template to download/upload

### ISSUE THREAD

**Issue by Sepehr-Sobhani** [bcgov/cas-registration#2426]
#### Dependencies

- Blocked by
- Blocking

#### Description:
Application template [If shipment is on or before March 31](https://www2.gov.bc.ca/assets/download/F5375D72BE1C450AB52C2E3E6A618959)

Application template [If shipment is on or after April 1](https://www2.gov.bc.ca/assets/download/751CDDAE4C9A411E974EEA9737CD42C6)

#### Acceptance Criteria:

**Scenario: Registering as a New Entrant**

*Given* that I am an industry user wanting to register my operation,  
*When* I select "New Entrant" as the purpose of registration,  
*Then* I can complete the application information for new entrants.

*Given* that I am on the new entrant application form,  
*When* I switch between different dates of first shipments,  
*Then* I receive different application templates for new entrants based on the selected option.

#### Development Checklist:

- [x] Update the placeholder link for `on_or_before_march_31_section` and `on_or_after_april_1_section` in `newEntrantOperation.ts` schema
- [x] Add frontend test
- [x] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

**Comment by andrea-williams** [comment]
@patriciarussellCAS and/or @zoeyli-46 - just flagging that right now we don't have the links for the templates that external users should be downloading for applying to be a new entrant - https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=13653-1955&m=dev&t=vnzNWN8SHSE5j38Y-1

**Comment by zoeyli-46** [comment]
I've just added them to the ticket

**Comment by Sepehr-Sobhani** [comment]
To access these files, we need to request permission. Is there a link that we can share without needing access? Or should we host the file ourselves?

**Comment by andrea-williams** [comment]
ooooh I don't think we want to be hosting the files ourselves - CAS needs to host that on their side so they can update them whenever they want

**Comment by zoeyli-46** [comment]
What does it require on their end to "host the files"? Does it go on an external website? Or is it uploaded somewhere that is a just a "private link" of some sort? 

Because when I asked Tiegan whether they can host it and provide us with the link, Tiegan says if they need to the application forms on the external CAS website and then link it in the app, then that kind of website updates is currently blocked due to the election, and they may not be able to put it up on the CAS site for a few months.
 
Though, I'm not sure if hosting the file equates to putting it up on an external website? @andrea-williams or @Sepehr-Sobhani if either of you know if its different?

**Comment by patriciarussellCAS** [comment]
Just updating this ticket with the decision that the BA will host the application template on the BC Gov website.

**Comment by zoeyli-46** [comment]
Engagement team has provided the template links hosted on their side. I've updated this ticket with the links and it is ready to be pulled in, should this go into (next) sprint backlog? @patriciarussellCAS

**Comment by patriciarussellCAS** [comment]
To consider at next backlog refinement.

**Comment by fviduya** [comment]
I have tested this in the DEV environment, [here is the test document for reference](https://bcgov-my.sharepoint.com/:w:/g/personal/francesca_viduya_gov_bc_ca/EULp5ZR8FD1Ere-HA0OVYKwBapRFY_mOsyQknNrShHpJ3w?e=tcNeii).

I can verify that:
- [x] When registering an operation and selecting ```New Entrant Operation``` as the purpose of registration, I am able to complete the application form for new entrants
- [x] When I switch between different dates of first shipments, I am able to receive different application templates for new entrants based on the selected option

In the document I attached, I found a bug where clicking ```Save & Continue``` from the Attachments page loops back to the Final Review page, that has been reported under [#526](https://github.com/orgs/bcgov/projects/123/views/20?filterQuery=-status%3Adone+523&pane=issue&itemId=95612980&issue=bcgov%7Ccas-reporting%7C526)

**Comment by patriciarussellCAS** [comment]
I have reviewed the test document and will move this to accepted. thanks!!
